ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,72 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+### Segment Routing Summary
+
+- SRTE is enabled.
+
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+
+### SRTE Policies
+
+| Endpoint | Color | Preference | Name | Description | Label Stack | Index  | Weight | Explicit Null |
+| -------- | ----- | ---------- | ---- | ----------- | ----------- | ------ | ------ | ------------- |
+{%         if router_traffic_engineering.segment_routing.policy_endpoint is arista.avd.defined %}
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoint | arista.avd.natural_sort %}
+{%                 for color in router_traffic_engineering.segment_routing.policy_endpoint[endpoint].color | arista.avd.natural_sort %}
+{%                     if router_traffic_engineering.segment_routing.policy_endpoint[endpoint].color[color].name is arista.avd.defined %}
+{%                         set name = router_traffic_engineering.segment_routing.policy_endpoint[endpoint].color[color].name %}
+{%                     else %}
+{%                         set name = "Not set" %}
+{%                     endif %}
+{%                     if router_traffic_engineering.segment_routing.policy_endpoint[endpoint].color[color].description is arista.avd.defined %}
```

Comment:
We tend to use "-" instead of "Not set" in our doc templates. There is also a way you could simplify and replace these if-else-clauses to something like this (with my suggested data model change included):

```jinja
...
{%                 for color in endpoint.colors %}
...
{%                     set description = color.description | arista.avd.default("-") %}
```

Same with set name, etc. This will make the template more concise and readable.